tests/snapshots/index.js.md

```diff
@@ -25,6 +25,20 @@ Generated by [AVA](https://avajs.dev).
 > stdout
 
     `owner and repositories set, creating token for repositories "create-github-app-token" owned by "actions"␊
+    ␊
+    GET /api/v3/repos/actions/create-github-app-token/installation␊
+    ␊
+    ␊
+    GET /api/v3/repos/actions/create-github-app-token/installation␊
+    ␊
+    ␊
+    POST /api/v3/app/installations/123456/access_tokens␊
+    "{\\"repositories\\":[\\"create-github-app-token\\"]}"␊
+    ␊
+    ␊
+    POST /api/v3/app/installations/123456/access_tokens␊
+    "{\\"repositories\\":[\\"create-github-app-token\\"]}"␊
+    ␊
```

Comment:
... however, now it looks like we are sending requests twice? This is worth investigating, next time

Reply:
nevermind, I looked into it and we do not send requests twice, I think `undici` invokes the `body()` and `path()` callbacks twice though, for some reason.

I worked around it in dd71e5594897ee755be7bdbad5730b1f28bfb2ce.

Ideally undici would give us a way to call a method whenever a request is intercepted, but I couldn't find such a thing yet. It would be much more elegant.

Reply:
figured it out 🥳